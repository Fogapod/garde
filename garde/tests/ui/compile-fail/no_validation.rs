#![allow(dead_code)]

#[derive(garde::Validate)]
struct Struct {
    field: u64,
}

#[derive(garde::Validate)]
struct Tuple(u64);

#[derive(garde::Validate)]
enum Enum {
    Struct { field: u64 },
    Tuple(u64),
}

fn main() {}
