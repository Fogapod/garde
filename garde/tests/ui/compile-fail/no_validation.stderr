error: field has no validation, use `#[garde(skip)]` if this is intentional
 --> tests/ui/compile-fail/no_validation.rs
  |
  |     field: u64,
  |            ^^^

error: field has no validation, use `#[garde(skip)]` if this is intentional
 --> tests/ui/compile-fail/no_validation.rs
  |
  | struct Tuple(u64);
  |              ^^^

error: field has no validation, use `#[garde(skip)]` if this is intentional
 --> tests/ui/compile-fail/no_validation.rs
  |
  |     Struct { field: u64 },
  |                     ^^^

error: field has no validation, use `#[garde(skip)]` if this is intentional
 --> tests/ui/compile-fail/no_validation.rs
  |
  |     Tuple(u64),
  |           ^^^
